src/Entities/Entity.cpp

```diff
@@ -943,6 +944,7 @@ void cEntity::HandlePhysics(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 		{
 			// Normal gravity
 			fallspeed = m_Gravity * DtSec.count();
+			NextSpeed -= NextSpeed * m_AirDrag * DtSec.count();
```

Comment:
I see, it's not exactly m_AirDrag / sec when compounded.  I'll modify it so it's m_AirDrag per interpolated tick.